blockchain/v0/reactor.go

```diff
@@ -178,7 +179,11 @@ func (bcR *BlockchainReactor) RemovePeer(peer p2p.Peer, reason interface{}) {
 func (bcR *BlockchainReactor) respondToPeer(msg *bcproto.BlockRequest,
 	src p2p.Peer) (queued bool) {
 
-	block := bcR.store.LoadBlock(msg.Height)
+	block, err := bcR.store.LoadBlock(context.TODO(), msg.Height)
+	if err != nil {
+		// todo(evan): handle this error
+		panic(err)
```

Comment:
There is a lot of self-reminders to handle the error instead of panicking. Do you have a vision of how those could be fixed? Do we need an issue for this or is it too small for that?